test/scripts/apidoc/examplesAndDeprecations.spec.ts

```diff
@@ -79,70 +61,74 @@ describe('examples and deprecations', () => {
     });
 
     // eslint-disable-next-line @typescript-eslint/no-misused-promises
-    it.each(Object.entries(methodsByName))('%s', async (methodName, method) => {
-      const signatures: SignatureReflection[] =
-        method.signatures || method.type?.['declaration'].signatures;
-      const signature = signatures[signatures.length - 1];
-
-      // Extract examples and make them runnable
-      let examples = extractRawExamples(signature).join('').trim() ?? '';
-      examples = examples.replace(
-        /faker([A-Z]{2})\./g,
-        (_, locale: string) => `faker.locale = '${locales[locale]}';\nfaker.`
-      );
-
-      expect(examples, `${moduleName}.${methodName} to have examples`).not.toBe(
-        ''
-      );
-
-      // Save examples to a file to run it
-      const dir = resolve(__dirname, 'temp', moduleName);
-      mkdirSync(dir, { recursive: true });
-      const path = resolve(dir, `${methodName}.ts`);
-      writeFileSync(
-        path,
-        `import { faker } from '../../../../../src';\n${examples}`
-      );
-
-      // Run the examples
-      await import(path);
+    it.each(Object.entries(methodsByName))(
+      '%s',
+      async (methodName, signature) => {
+        // Extract examples and make them runnable
+        let examples = extractRawExamples(signature).join('').trim() ?? '';
```

Comment:
This looks like `??` is not necessary here :thinking: 
The lhs has no nullish value, otherwise the join or trim would already result in "NPE"
I assume we can omit the rhs

```suggestion
        let examples = extractRawExamples(signature).join('').trim();
```

If this fails, I'm very interested in why :slightly_smiling_face:

Reply:
Fixed